web/client/translations/data.it-IT.json

```diff
@@ -2257,8 +2257,8 @@
             "addMarkRule": "Aggiungi regola marker",
             "shape": "Forma",
             "fill": "Colore riempimento",
-            "clampOutlineToGround": "Adattare il contorno alla superficie",
             "clampToGround": "Adattarsi alla superficie",
+            "clampOutlineToGround": "Adattarsi alla superficie",
```

Comment:
```suggestion
            "clampOutlineToGround": "Adattare alla superficie",
```

Reply:
ok i'll include this change also in other translations